### PR TITLE
[ new ] more interleaving functions in Codata.Stream

### PR DESCRIPTION
**Pull request by gallais**
Came across this cute `cantor` function while formalising Liam's paper in
Idris so thought I'd see how it fairs with sized codata.

Also threw in `interleave⁺` which is very easy 	to define with sized types :)

### CODE REVIEW

File: src/Codata/Stream.agda [hunk 1]
```diff
@@ -13,89 +13,159 @@ open import Codata.Thunk as Thunk using (Thunk; force)
...
+drop : ℕ → Stream A ∞ → Stream A ∞
+drop n xs = proj₂ (splitAt n xs)
 
-module _ {a} {A : Set a} where
+chunksOf : (n : ℕ) → Stream A ∞ → Stream (Vec A n) ∞
+chunksOf n = chunksOfAcc n id module ChunksOf where
 
-  iterate : (A → A) → A → Stream A ∞
-  iterate f = unfold < f , id >
+  chunksOfAcc : ∀ {i} k (acc : Vec A k → Vec A n) →
```
Comment (MatthewDaggitt) [inline src/Codata/Stream.agda:115]: Can this `i` be left to be a variable?

File: CHANGELOG.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ Other minor additions
   ```agda
   *-pres-∣ : o ∣ m → p ∣ n → o * p ∣ m * n
   ```
-  
+
```
Comment (MatthewDaggitt) [inline CHANGELOG.md:216]: Why is your `fix-whitespace` picking up whitespace in the CHANGELOG when it's supposed to be ignored?

https://github.com/agda/agda-stdlib/blob/911b52d54d39939a78ba63a387e30cc90ad64163/fix-whitespace.yaml#L12

**Comment by MatthewDaggitt** [comment]
Otherwise, looks very cool!

File: CHANGELOG.md (hunk 2 above)
Comment (gallais) [reply CHANGELOG.md:216]: My emacs is configured to automatically fix whitespace issues. :x

**Pull request merged**